packages/sewing-kit-koa/src/middleware.ts

```diff
@@ -17,7 +17,10 @@ export interface Options {
   manifestPath?: string;
 }
 
-const ASSETS = Symbol('assets');
+// Have to cast as `any` or typescript will complain
+// about using a "unique symbol as an index type".
+// https://github.com/microsoft/TypeScript/issues/1863
```

Comment:
I don't understand, what changed that caused this?

Reply:
@lemonmade the change from `types/koa` from `2.0.49 => 2.0.50`. My guess is this change https://github.com/DefinitelyTyped/DefinitelyTyped/commit/80c6d507792b9706617ca27e4fa5ec9e314db8a5#diff-7605d695f1da86ef8f6a929780113155R724

Reply:
`State` is no longer `any`

Reply:
Can we cast `ctx.state` as `any` then? I don't like a type that is a straight up lie when we could switch to looser typing in one spot instead.